projects/components/src/table/cells/data-renderers/table-data-cell-renderer.component.ts

```diff
@@ -117,6 +111,6 @@ export class TableDataCellRendererComponent implements OnInit {
   }
 
   public parseValue(): unknown {
-    return this.componentRef?.instance.parseValue(this.cellData, this.rowData);
+    return new this.columnConfig!.parser!().parseFilterValue(this.cellData);
```

Comment:
I'll keep saying it - every time you have to use `!`, it's indicative of a problem.